app/app/crashes/[record_locator]/page.tsx

```diff
@@ -165,6 +166,15 @@ export default function CrashDetailsPage({
           />
         </Col>
       </Row>
+      <Row>
+        <Col sm={12} md={6} className="mb-3">
```

Comment:
i went half-width with this card, thinking that we might put the crash notes component next to it. this would be a change from the old VZE where both the recommendations and notes sections are full-width on all display sizes.